src/main/java/edu/kis/powp/jobs2d/drivers/SelectworkspaceMenuOptionListener.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.canva.WorkspaceCanva;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectworkspaceMenuOptionListener implements ActionListener {
+    private DriverManager driverManager;
+    private Job2dDriver driver = null;
+    private WorkspaceCanva canvas = null;
+
+    public SelectworkspaceMenuOptionListener(WorkspaceCanva canva, DriverManager driverManager) {
+        this.driverManager = driverManager;
+        this.driver = canva;
+        this.canvas = canva;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        driverManager.setCurrentDriver(driver);
```

Comment:
I think here is the biggest issue. The current driver should be selected via drivers menu and ignoring&overriding the current driver setting is confusing. 
The solution should always consider the driver selected by the user of the application.